src/main/pages/che-6/user-guide/ide-projects.adoc

```diff
@@ -10,25 +10,26 @@ folder: che-6/user-guide
 
 
 [id="default-location-of-projects"]
-== Default Location of Projects
+== Default location of projects
 
-When a workspace machine with a workspace agent starts, the IDE makes calls to Project API to get workspace projects. By default, they are located in `projects` in a machine container/pod. This location isn’t configurable as many other IDE components depend on it.
+When a workspace machine with a workspace agent starts, the IDE makes calls to the project API to get workspace projects. By default, projects are located in the `/projects` directory in a machine container or pod. You can not configure this location because other IDE components depend on it.
 
-Projects are displayed in the Project Tree on the left. You can also see them in the terminal - `ls -la /projects`. When a workspace is stopped, its machines are terminated, however, `/projects` gets mounted onto the host file System (either data volume or PVC), so when a new container/pod is started, your workspace projects are always there.
+Projects are displayed in the project tree in the left panel. To view the projects on the terminal, use the `ls -la /projects` command. When a workspace is stopped, its machines are terminated. The `/projects` directory gets mounted on the host file system (either data volume or PVC). When a new container or pod is started, your workspace projects are always there.
 
 image::ide/project_tree.png[]
 
 [id="file-watchers"]
-== File Watchers
+== File watchers
 
-Since changes to project source files may not be originated from IDE editor only, there are file watchers listening to changes on the file system. Thus, if you create, edit or delete resources on your file system using terminal or REST API, changes will be propagated to project tree and editor. File watchers are disabled in certain situations, like project import, to avoid flooding events.
+Since changes to project source files may not be originated from the IDE editor only, file watchers listen to changes on the file system. If you create, edit, or delete resources on your file system using the terminal or REST API, these changes are propagated to the project tree and the editor. When importing projects, file watchers are disabled to avoid flooding events.
```

Comment:
The opening sentence is long and contains information that I don't believe is necessary. We can remove everything before the comma and retain the important information without it. If I'm mistaken and the information _is_ necessary, we can make the sentence shorter while conveying the same information.

Reply:
I changed it to "Changes to project source files do not originate from the IDE editor only. File watchers listen to changes on the file system. Resources that you create, edit, or delete on your file system using the terminal or REST API propagate to the project tree and the editor. When importing projects, file watchers are disabled to avoid flooding events."

Reply:
I think that's much clearer. Thank you! :+1: